examples/demo.js

```diff
@@ -8,7 +8,27 @@ var defaultHtmlSequencerUi = require('./lib/defaultHtmlSequencerUi.js'),
 window.onload = function () {
   sequencer = ImageSequencer();
 
-  function refreshOptions() {
+  options = {
+      sortField: 'text',
+      openOnFocus: false,
+      onInitialize: function () {
+          var that = this;
+          this.$control.on("click", function () {
+              that.ignoreFocusOpen = true;
+              setTimeout(function () {
```

Comment:
Why is this delay added? Isn't there a better method of doing this(just asking). What exactly does it do anyway?

Reply:
The actual time difference between the click and onFocus is not controlled by the user and is usually about 2-5 milliseconds. Waiting for 50 is more than enough to prevent unexpected behavior.

Reply:
So the dropdown opens when `onFocus` is triggered?

Reply:
Then what happens `onClick`?

Reply:
When the input is not focused and you click, first fires the onFocus event and then a click event on this.$control
When you click on the input while it is focused and the dropdown is shown, it's the opposite - first fires onBlur, then click and finally onFocus

Reply:
I was asking what each event does. Not what triggers it.

Reply:
What happens when an event is triggered? What does it do?

Reply:
These events used for correct closing dropdown when we clicking outside the text but still in the box of dropdown. We get the same functionality as with openOnFocus set to true, except we don't get this strange unable-to-close behavior.

>So the dropdown opens when onFocus is triggered?
Yes, but only after 50 miliseconds of clicking on area of dropdown to protect from reopening

Reply:
Ok great.